src/components/HomepageFeatures/index.js

```diff
@@ -42,34 +42,54 @@ const FeatureList = [
         Svg: require('@site/static/img/undraw_docusaurus_tree.svg').default,
         description: (
             <>
-                Learn about our coding standards, styles, and conventions.
+                Read our style guidelines, including coding standards and naming conventions.
             </>
         ),
-        linkText: 'Learn more',
+        linkText: 'View Standards',
     },
     {
         link: '/docs/apis',
-        title: 'Read our API Guides',
+        title: 'API guides',
         Svg: require('@site/static/img/undraw_docusaurus_react.svg').default,
         description: (
             <>
-                Want to learn more about our APIs?
-                <br />
-                Learn about them through our series of deep-dive guides, complete with examples and explanations.
+                Learn all about Moodle APIs with our deep-dive guides, complete with examples and explanations.
             </>
         ),
-        linkText: 'Read API Guides',
+        linkText: 'Read API guides',
     },
     {
         link: '/general/community',
-        title: 'Join our community',
+        title: 'Developer community',
+        Svg: require('@site/static/img/undraw_docusaurus_react.svg').default,
+        description: (
+            <>
+                Join the open source community that makes Moodle.
+            </>
+        ),
+        linkText: 'Discover communities',
+    },
+    {
+        link: '/general/releases',
+        title: 'Release notes',
         Svg: require('@site/static/img/undraw_docusaurus_react.svg').default,
         description: (
             <>
-                Discover our community, and learn about our mission, and our roadmap.
+                Technical specs and new features of Moodle releases, supported versions and more.
             </>
         ),
-        linkText: 'Discover Communities',
+        linkText: 'Check the Release notes',
```

Comment:
```suggestion
        linkText: 'Check the release notes',
```

Reply:
I disagree with this one. "Release notes" are a proper noun, they're not just another word. They should therefore be capitalised.

Reply:
Done.